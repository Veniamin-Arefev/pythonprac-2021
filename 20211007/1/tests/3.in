(93, 93), (52, 19), (44, 85), (49, 30), (76, 21), (14, 83), (86, 63), (74, 49), (78, 27), (97, 85), (4, 98), (32, 98), (27, 1), (93, 69), (97, 63), (11, 24), (90, 75), (2, 26), (70, 32), (93, 39)