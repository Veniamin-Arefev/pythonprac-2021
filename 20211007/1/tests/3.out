((93, 93), (97, 85), (32, 98))
